From 6004870d53221caf05a330fcfc8a72e68b2b8898 Mon Sep 17 00:00:00 2001
From: Dave Airlie <airlied@redhat.com>
Date: Tue, 06 Aug 2013 03:03:18 +0000
Subject: configure.ac: fix dri enabled with kms only picked

Otherwise we'd fail saying DRI1 wasn't possible, when that
is exactly what we asked for.

[ickle: The breakage was introduced with

commit bd6ffd1ad2f903215d2c631252e742a0f6893054 [2.21.14]
Author: Chris Wilson <chris@chris-wilson.co.uk>
Date:   Sat Jul 27 15:33:19 2013 +0100

    configure: Print a summary of compilation options

which modified the search to only take place if UMS was enabled, but
missed mollifying the resulting error.]

Signed-off-by: Dave Airlie <airlied@redhat.com>
---
diff --git a/configure.ac b/configure.ac
index 104113e..88f29cc 100644
--- a/configure.ac
+++ b/configure.ac
@@ -433,7 +433,7 @@ else
         DRI1_CFLAGS=""
         DRI1_LIBS=""
 
-        if test "x$enable_dri" = "xyes"; then
+        if test "x$enable_dri" = "xyes" -a "x$KMS_ONLY" != "xyes"; then
                 AC_MSG_ERROR([DRI1 requested but prerequisites not found])
         fi
 fi
--
cgit v0.9.0.2-2-gbebe
